react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -216,12 +216,14 @@ AllTrajectoryHidden.args = {
     layers: [
         {
             ...defaultProps.layers[0],
-            lineStyle: { color: [0, 0, 0, 0] },
+            lineStyle: { color: [100, 100, 100, 100] },
+            wellHeadSyle: { pointRadiusScale: 0 },
```

Comment:
pointRadiusScale can be a callback function too, similar to lineStyle.
With that we will be able to hide the wellhead selectively.
Example, wellHeadStyle: {size?: callback | number, color?: callback | RGBAColor }